<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SVET" failures="1" tests="2" name="Test" time="21.411" errors="0" timestamp="2022-05-30T10:09:54 IST">
  <testcase classname="Sampmavan.LoginpageobjectTest" name="SuccussfullLogin" time="3.031">
    <failure type="java.lang.AssertionError" message="expected [Login incorrect. Please try ] but found [Login incorrect. Please try again]">
      <![CDATA[java.lang.AssertionError: expected [Login incorrect. Please try ] but found [Login incorrect. Please try again]
at Sampmavan.LoginpageobjectTest.SuccussfullLogin(LoginpageobjectTest.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- SuccussfullLogin -->
  <testcase classname="Sampmavan.LoginpageobjectTest" name="forgetpassword" time="2.006"/>
</testsuite> <!-- Test -->
